src/shared/constants/routes.ts

```diff
@@ -119,7 +119,7 @@ const ROUTES = {
       flowId: string | null;
       publicAppletKey: string | null;
     }) =>
-      `/protected/answer-processing?appletId=${appletId}&eventId=${eventId}&activityId=${activityId}&flowId=${flowId}&publicAppletKey=${publicAppletKey}`,
+      `/protected/answer-processing?appletId=${appletId}&eventId=${eventId}&activityId=${activityId}${flowId ? `flowId=${flowId}` : ''}${publicAppletKey ? `publicAppletKey=${publicAppletKey}` : ''}`,
```

Comment:
Just a comment (no need to do it now): Maybe it would be a good idea to somehow utilize the [Tagged templated](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates) to avoid multiple conditional statements in template strings.

Reply:
Yeah, maybe it makes sense to think how I can improve the readability of this part of the code